### PR TITLE
pkp/pkp-lib#1709 provide export XML for download

### PR DESCRIPTION
**Pull request by bozana**
S. i) here: https://github.com/pkp/pkp-lib/issues/1709
(This PR considers also other export plugins)

### CODE REVIEW

File: classes/plugins/PubObjectsExportPlugin.inc.php [hunk 1]
```diff
@@ -163,8 +163,13 @@ function executeExportAction($request, $objects, $filter, $tab, $objectsFileName
 			assert($filter != null);
 			// Get the XML
 			$exportXml = $this->exportXML($objects, $filter, $context);
-			header('Content-type: application/xml');
-			echo $exportXml;
+			$exportFileName = $this->getExportFileName($objectsFileNamePart, $context);
+			file_put_contents($exportFileName, $exportXml);
```
Comment (asmecher) [inline classes/plugins/PubObjectsExportPlugin.inc.php]: Some of this is starting to reinvent the TemporaryFileManager; I haven't worked with that class for a while, but I think it should be pretty quick to make use of it instead for management of file creation/deletion/etc.

**Comment by bozana** [comment]
@asmecher, I did the change, to use TemporaryFileManager everywhere. Could you please take a look again (the last commit)? THANKS!!!

**Comment by bozana** [comment]
@asmecher, I now changed the commit above to use just the normal FileManager...

**Comment by asmecher** [comment]
Merged, thanks!

**Pull request merged**